src/Framework/FileMock.php

```diff
@@ -40,16 +48,51 @@ public static function create($content, $extension = NULL)
 	}
 
 
+	public static function register()
```

Comment:
Purpose: when someone want to use mock:// before `FileMock::create()`, e.g. testing with `mock://not-exist-yet`.